### PR TITLE
add zh trans content/zh/docs/reference/setup-tools/kubeadm/generated/…

### PR DESCRIPTION
**Pull request by tanjunchen**

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset_phase_update-cluster-status.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+<!--
+### Synopsis
+-->
+### 概要
+
+
+<!--
+Remove this node from the ClusterStatus object if the node is a control plane node.
+-->
+如果该节点是控制平面节点，请从 ClusterStatus 对象中删除该节点。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset_phase_update-cluster-status.md]:
```suggestion
如果该节点是控制平面节点，从 ClusterStatus 对象中删除该节点。
```

**Comment by tanjunchen** [comment]
@tengqm 已修改

**Pull request merged**